Instagram_App/Instagram_App/ViewControllers/search/CollectionViewCell.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  CollectionViewCell.swift
+//  Instagram_App
+//
+//  Created by coder on 23.10.2022.
+//
+
+import UIKit
+
+/// ячейка collection view
+final class PostCollectionViewCell: UICollectionViewCell {
+    
+    // MARK: IBOutlet
+    @IBOutlet weak var postImageView: UIImageView!
+    
+    // MARK: Public Methods
+    func configCell(_ model: PostCell, index: Int) {
+        postImageView.image = UIImage(named: model.nameImage[index])
```

Comment:
```suggestion
        postImageView.image = UIImage(named: model.nameImage[index])
```
либо переделай _ model: PostCell на name: String и убрать индекс 
либо сделай безопасное извлечение